CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Compatibility with `xarray>=2025.03`.
 
+### Fixed
+- Fixed scaling for adjoint sources when differentiating with respect to field monitor data to account for the mesh size of the monitor and thus the created source. This aligns adjoint gradient magnitudes with numerical finite difference gradients for field data.
```

Comment:
wouldn't hurt to just replace "field monitor data" with `FieldData` to be explicit.